### ISSUE TITLE
ModuleNotFoundError: No module named 'app' : flask db migrate

### ISSUE THREAD

**Issue by rlabrador** [nickjj/flask-db#11]
Hello.
When launching the command : flask db migrate
'I went all throw the process of updating the versioned script by updating upgrade and dowgrade functions...

Output is :
Traceback (most recent call last):
  File "/home/rlab/anaconda3/envs/backend-env/bin/alembic", line 8, in <module>
    sys.exit(main())
  File "/home/rlab/anaconda3/envs/backend-env/lib/python3.9/site-packages/alembic/config.py", line 581, in main
    CommandLine(prog=prog).main(argv=argv)
  File "/home/rlab/anaconda3/envs/backend-env/lib/python3.9/site-packages/alembic/config.py", line 575, in main
    self.run_cmd(cfg, options)
  File "/home/rlab/anaconda3/envs/backend-env/lib/python3.9/site-packages/alembic/config.py", line 552, in run_cmd
    fn(
  File "/home/rlab/anaconda3/envs/backend-env/lib/python3.9/site-packages/alembic/command.py", line 298, in upgrade
    script.run_env()
  File "/home/rlab/anaconda3/envs/backend-env/lib/python3.9/site-packages/alembic/script/base.py", line 489, in run_env
    util.load_python_file(self.dir, "env.py")
  File "/home/rlab/anaconda3/envs/backend-env/lib/python3.9/site-packages/alembic/util/pyfiles.py", line 98, in load_python_file
    module = load_module_py(module_id, path)
  File "/home/rlab/anaconda3/envs/backend-env/lib/python3.9/site-packages/alembic/util/compat.py", line 184, in load_module_py
    spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "db/env.py", line 7, in <module>
    from app import create_app
ModuleNotFoundError: No module named 'app'

Any ideas?
Thanks a lot in advance.

**Comment by rlabrador** [comment]
Thanks for your help.
Really weird...
Assuming doing a bad thing but don't understand... ;-(

![Screen Shot 01-21-22 at 12 42 AM 001](https://user-images.githubusercontent.com/7700620/150439834-1aaf9076-1ab1-43bf-9b39-9828406a15f7.PNG)
and

![Screen Shot 01-21-22 at 12 41 AM](https://user-images.githubusercontent.com/7700620/150440009-cc487868-4ffd-4736-b6f9-2a8c8adc046b.PNG)

**Comment by nickjj** [comment]
It looks like you need to do `from app.app import create_app` because you have an app folder with your app.py file in that folder.

**Comment by rlabrador** [comment]
Nop... Doesn't work neither... :-(

Error: While importing 'app.app', an ImportError was raised.

![Screen Shot 01-21-22 at 09 33 AM](https://user-images.githubusercontent.com/7700620/150493722-404fd896-8290-47d3-a2b2-bb171cb037ad.PNG)

**Comment by rlabrador** [comment]
Running debugging mode ...
No module named 'app'
  File "/d/dev_/py/waspy/wasptracker0/backendapp2/db/env.py", line 13, in <module>
    from app.app import create_app

![Screen Shot 01-21-22 at 09 35 AM](https://user-images.githubusercontent.com/7700620/150493941-5b7a1578-dbf5-40b6-a577-a384451bd1d2.PNG)

**Comment by nickjj** [comment]
Did you follow these steps from the readme? https://github.com/nickjj/flask-db#ensuring-the-db-command-is-available-and-everything-works

What is your `FLASK_APP` set to?

**Comment by rlabrador** [comment]
Super. Your insights put me ont the good track.

I still had a conflict between the standard command : python app/app.py and flask run, shell or db that warned saying the app module wasn't found.
I realized PYTHONPATH wasn't set neither.

I did a bit of cleaning. 
Moved up app.py to root folder.
set PYTHONPATH to PYTHONPATH="." 
changed FLASK_APP="app/app.py" to FLASK_APP="app.py"
renamed accordingly some imports of my apis.
flask db, shell and run works fine as well as well my apis.
Perfect time for seeding the db now. :-)

Thanks again for maintaining this great package to the community.
Richard




Thanks a lot for your time.
